integration_tests/fixtures/forseti_integration_testing/main.tf

```diff
@@ -0,0 +1,110 @@
+/**
```

Comment:
Please shorten the path here to `integration_tests/fixtures/testing/main.tf`.  Because this is a testing fixture.

Reply:
Renamed to `integration_tests/fixtures/forseti/main.tf` as it needs to match with `integration_tests/fixtures/forseti`.